server/src/main/java/org/apache/calcite/avatica/server/AbstractAvaticaHandler.java

```diff
@@ -34,6 +38,7 @@
  */
 public abstract class AbstractAvaticaHandler extends AbstractHandler
     implements MetricsAwareAvaticaHandler {
+  private static final Logger LOG = LoggerFactory.getLogger(AbstractAvaticaHandler.class);
```

Comment:
nit: Looks like LOG is potentially unused?

Reply:
ah, yup. I had some logging in here as a part of my work, but considered none of it beneficial long-term.